modules/deploy/pages/deployment-option/cloud/security/cloud-security-network.adoc

```diff
@@ -176,3 +176,20 @@ address range of the destination VPC.
 It is strongly recommended to reject all network traffic initiated from a Redpanda
 Cloud network and only accept traffic from Kafka connectors that connect to
 your internal data stores to retrieve or push data.
+
+== Azure network services
+
+Redpanda Cloud supports the following Azure network services:
+
+* Time synchronization
++
+To ensure time synchronization, Redpanda Cloud in Azure uses https://learn.microsoft.com/en-us/azure/virtual-machines/windows/time-sync[Microsoft time sync^].
```

Comment:
maybe we should use the Linux counterpart? https://learn.microsoft.com/en-us/azure/virtual-machines/linux/time-sync

Reply:
Thanks @c4milo! I've incorporated your feedback. Please give it all a final review!